aspnetcore/migration/90-to-100/includes/blazor.md

```diff
@@ -3,3 +3,30 @@ Complete migration coverage for Blazor apps is scheduled for September and Octob
 ### Adopt passkey user authentication in an existing Blazor Web App
 
 For guidance, see <xref:security/authentication/passkeys/blazor?pivots=existing-app>.
+
+### Update the `IdentityRedirectManager` in apps based on the Blazor Web App template with Individual Accounts
+
+The `IdentityRedirectManager` previously threw an <xref:System.InvalidOperationException> in the `RedirectTo` method to ensure the method wasn't called from an interactive render mode and all the redirection methods were marked with the [`[DoesNotReturn]` attribute](xref:System.Diagnostics.CodeAnalysis.DoesNotReturnAttribute). For more information, see <xref:aspnetcore-10#navigationmanagernavigateto-no-longer-throws-a-navigationexception>.
```

Comment:
We decided to make the new behavior that doesn't throw on navigation opt-in, so users only need to do these migration steps if they choose to set `    <BlazorDisableThrowNavigationException>true</BlazorDisableThrowNavigationException>`